python/cudf/cudf/core/frame.py

```diff
@@ -4718,17 +4729,8 @@ def shape(self):
         return (len(self),)
 
     def __iter__(self):
-        """
```

Comment:
Do we want to remove this docstring?

Reply:
I think it's fine either way. I can add it back if you prefer. I think this behavior would be cleaner to move to a mixin anyway, in which case it would be much more self-documenting in the long term and easier to add to different classes.

Reply:
Discussed offline, I added it back to be safe.